src/test/java/io/zipcoder/interfaces/TestInstructor.java

```diff
@@ -0,0 +1,72 @@
+package io.zipcoder.interfaces;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TestInstructor {
+
+	@Test
+	public void testImplementation() {
+		Instructor Leon = new Instructor("");
+		Assert.assertTrue(Leon instanceof Teacher);
+		
+	}
+	
+	@Test
+	public void testInheritance() {
+		Instructor Leon = new Instructor("");
+		Assert.assertTrue(Leon instanceof Person);
+	}
+	
+	@Test
+	public void testTeach()
+	{
+		//Given
```

Comment:
Given
`someStudent`
`someStudyTime`

When
`someStudent` is taught for `someStudyTime`

Then
`someStudent`'s study time should be `someStudyTime`.